crates/driver/src/domain/competition/solution/fee.rs

```diff
@@ -146,7 +238,7 @@ impl Fulfillment {
         let fee_from_volume = self.fee_from_volume(prices, max_volume_factor)?;
         // take the smaller of the two
         let protocol_fee = std::cmp::min(fee_from_surplus, fee_from_volume);
-        tracing::debug!(uid=?self.order().uid, ?fee_from_surplus, ?fee_from_volume, ?protocol_fee, executed=?self.executed(), surplus_fee=?self.surplus_fee(), "calculated protocol fee");
+        tracing::debug!(?fee_from_surplus, ?fee_from_volume, ?protocol_fee, executed=?self.executed(), surplus_fee=?self.surplus_fee(), "calculated protocol fee");
```

Comment:
order uid is very important information to have here. And given that JIT orders should not have fees in the first place it's not great that I got removed here.